platform/darwin/src/MGLTileSet.h

```diff
@@ -11,43 +11,43 @@ typedef NS_ENUM(NSUInteger, MGLTileSetScheme) {
 
 /**
  The `MGLTileSet` class holds the tile URL template strings and associated
- configuration for those strings. It can be passed to a `MGLVectorSource` or
+ configuration for those strings. It can be passed to an `MGLVectorSource` or
  `MGLRasterSource` instead of an `NSURL` representing a TileJSON URL to create a 
  source.
  */
 @interface MGLTileSet : NSObject
 
 /**
- A REQUIRED `NSArray` of `NSString` objects that represent the tile templates.
+ An `NSArray` of `NSString` objects that represent the tile templates.
  */
 @property (nonatomic, copy) NS_ARRAY_OF(NSString *) *tileURLTemplates;
 
 /**
- An OPTIONAL `NSNumber` object containing an integer; specifies the minimum zoom
- level at which the source will display tiles. The value should be in the range 
- of 0 to 22. The default value is 0 and the source will use the default value
+ An `NSNumber` object containing an integer; specifies the minimum zoom level at 
+ which the source will display tiles. The value should be in the range of 0 to 
+ 22. The default value is 0 and the source will use the default value
  if `minimumZoomLevel` is nil;
  */
 @property (nonatomic, nullable) NSNumber *minimumZoomLevel;
 
 /**
- An OPTIONAL `NSNumber` object containing an integer; specifies the maximum zoom
- level at which to display tiles. The value should be in the range of 0 to 22
- and greater than `minimumZoomLevel`. The default value is 22 and the source
- will use the default value if `maximumZoomLevel` is nil;
+ An `NSNumber` object containing an integer; specifies the maximum zoom level at 
+ which to display tiles. The value should be in the range of 0 to 22 and greater 
+ than `minimumZoomLevel`. The default value is 22 and the source will use the 
+ default value if `maximumZoomLevel` is nil;
  */
 @property (nonatomic, nullable) NSNumber *maximumZoomLevel;
 
 /**
- An OPTIONAL `NSString` object; Contains an attribution to be displayed
- when the map is shown to a user. The default value is nil;
+ An `NSString` object; Contains an attribution to be displayed when the map is 
```

Comment:
Nit: replace the semicolon with “that” and lowercase “contains” (plus one more below).